### ISSUE TITLE
JOOMMF metapackage printing all dependencies

### ISSUE THREAD

**Issue by marijanbeg** [joommf/joommf#79]
@fangohr wrote: Idea for meta-package joommf: I guess the joommf packge should have an overview of all required packages that are installed, maybe being able to print a list of them, together with the installed version on the local system, or a ‘missing’ if the package is not there. This is useful for debugging.

**Comment by marijanbeg** [comment]
I agree that this could be useful, but the problem is that we intended to use `joommf` package only as a metapackage for installation via `pip` and said that we do not want to allow `import joommf`. Any thoughts how this issue should be implemented?

**Comment by fangohr** [comment]
Sorry, that's probably what I tried to say. Here is a clearer version: I'd be happy to have a `joommf` module that can be imported (and thus can provide the functionality we discuss above, for example).

We can think of a number of convenient imports into the joommf namespace (from `discretisedfield`, `oommfc`, `micromagneticmodel` etc). What I was trying to say is that we should not rush to create those "convenience imports" but wait first and see what is really useful before implementing this. The reason being that once we have provided such short cuts, users will complain if we remove them again.
